imap_processing/tests/hi/test_hi_l1c.py

```diff
@@ -44,3 +49,30 @@ def test_empty_pset_dataset():
     assert dataset.spin_angle_bin.size == 3600
     assert dataset.esa_energy_step.size == n_esa_steps
     assert dataset.calibration_prod.size == n_calibration_prods
+
+
+@pytest.mark.parametrize("sensor_str", ["90sensor", "45sensor"])
+@mock.patch(
+    "imap_processing.hi.l1c.hi_l1c.frame_transform", side_effect=lambda a, b, c, d: b
+)
```

Comment:
Just checking, this is mocking the frame transform to simply return the input position? Maybe add a comment or replace argument b with position.